tests/test_ro_crate_parser.py

```diff
@@ -0,0 +1,137 @@
+# pylint: disable=missing-docstring
+# nosec assert_used
+from pathlib import Path
+
+import src.utils.filesystem.filters as filters
+from src.blueprints.datafile import RawDatafile
+from src.blueprints.dataset import RawDataset
+from src.blueprints.experiment import RawExperiment
+from src.blueprints.project import RawProject
+from src.extraction.manifest import IngestionManifest
+from src.profiles.ro_crate.ro_crate_parser import ROCrateParser
+from tests.fixtures.fixtures_ro_crate import (
+    fakecrate_root,
+    fixture_fake_ro_crate,
+    fixture_ro_crate_name,
+    fixture_rocrate_datafile,
+    fixture_rocrate_dataset,
+    fixture_rocrate_experiment,
+    fixture_rocrate_parser,
+    fixture_rocrate_project,
+    fixture_rocrate_uuid,
+    ro_crate_dataset_dir,
+    ro_crate_unlisted_file_dir,
+    rocrate_profile_json,
+    test_rocrate_content,
+)
+
+
+def test_load_crate(
+    fixture_rocrate_parser: ROCrateParser,
+    fakecrate_root: Path,
```

Comment:
Is this unused? Same for `ro_crate_unlisted_file_dir`

Reply:
Oh yeah they are I think that's the result of an inital misunderstanding of how fixtures work. I misread some docs and though they had to be specifically requested if other fixtures depended on them or their tmp files in addition to importing all of them or specifying in conftest.

Reply:
Oh yep, makes sense